### PR TITLE
Updated express-graphql and deps

### PR DESCRIPTION
**Pull request by yanickrochon**
Long needed update. All tests passing.

### CODE REVIEW

**Comment by yanickrochon** [comment]
@chentsulin could it be possible to have some feeback from you, or possibly allow contributors to actually contribute to this project, please? I'm hitting a bug and my solution is to either for you to merge and update this on npm, me to clone and create another package on npm (you know, duplication pollution of the ecosystem...) or switch to express.

I don't want the latter because this would mean days of refactoring the app.

The easiest solution would be to have contributors have access to this repository and publishing on npm. Ideally, this package should ideally be moved to the [GraphQL](https://github.com/graphql) organization.

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ language: node_js
 
 # https://github.com/nodejs/Release
 node_js:
-  - '11'
+  - '12'
```
Comment (sibelius) [inline .travis.yml:7]: should we add 14?

Comment (yanickrochon) [reply .travis.yml:7]: Sure. Should we remove Node v8 ? :) It gives errors and it's oddly outdated.

Comment (sibelius) [reply .travis.yml:7]: yep, let's deprecated it

**Comment by MaxDesiatov** [comment]
Can this be merged now?

**Pull request merged**